scripts/surface_group_pipeline_V1.m

```diff
@@ -0,0 +1,137 @@
+function surface_group_pipeline_V1()
+ % Example for the template- and surface-based full pipeline, using the
+ % function NST_PPL_SURFACE_TEMPLATE_V1.
+ %
+ % This script downloads some sample data of 10 "dummy" subjects (27 Mb), 
+ % as well as the Colin27_4NIRS template (19 Mb) if not available.
+ % All downloaded data will be stored in .brainstorm/defaults/nirstorm
+ %
+ % Data are imported in brainstorm in a dedicated protocol, using specific 
+ % naming conventions specific to NST_PPL_SURFACE_TEMPLATE_V1.
+ % Preprocessings and processings are then run up to group-level analysis:
+ %          1) Resampling to 5hz
+ %          2) Bad channels detection
+ %          3) Conversion to delta optical density
+ %          4) High pass filter
+ %          5a) Compute head model using fluences   
+ %          5b) Projection on the cortical surface using head model
+ %          6) 1st level GLM with pre-coloring
+ %          7) group-level GLM with MFX contrast t-maps
+ %
+ % This script illustrates a minimal fully functional analysis pipeline that can
+ % serve as a basis for another custom study.
+ %
+ % For a more detailed description, see the wiki page:
+ % https://github.com/Nirstorm/nirstorm/wiki/%5BWIP%5D-GLM-pipeline:-surface-and-template-based
+ % 
+ % For a more comprehensive example script, see:
+ % surface_template_full_group_pipeline_V1_all_opts.m
+ 
+%% Setup brainstorm
+if ~brainstorm('status')
+    % Start brainstorm without the GUI if not already running
+    brainstorm nogui
+end
+
+%% Check Protocol
+
+protocol_name = 'SurfaceGroupPipelineV1';
+
+if isempty(bst_get('Protocol', protocol_name))
+    gui_brainstorm('CreateProtocol', protocol_name, 1, 0); % UseDefaultAnat=1, UseDefaultChannel=0
+end
+
+
+% Set template for default anatomy
+nst_bst_set_template_anatomy('Colin27_4NIRS_Jan19');
```

Comment:
Should not be needed

Reply:
I thought the same thing but it is in fact needed for the group analysis.  Doing that allow us to project the output of the GLM onto the template surface. That's also why we use  `UseDefaultAnat=1` when creating the protocol.